utils/src/foreign_field.rs

```diff
@@ -182,10 +218,10 @@ pub trait BigUintForeignFieldHelpers {
     /// 2
     fn two() -> Self;
 
-    /// 2^{LIMB_SIZE}
+    /// 2^{LIMB_BITS}
```

Comment:
The names are now misleading as some functions use `LIMB_BITS = 88`, defined at the top level of the module
I would rename (or gather in a `kimchi` nested module) the functions depending on the 88-bits limb encoding.